docs/network-requirements.md

```diff
@@ -0,0 +1,16 @@
+# Network Requirements
+
+Antrea has a few network requirements to get started, ensure that your hosts and firewalls allow the necessary traffic based on your configuration.
+
+| Configuration              | Host(s)                          | ports/protocols   |
+| -------------------------- | -------------------------------- | ----------------- |
+| Antrea with VXLAN enabled  | All                              | UDP 4789          |
+| Antrea with Geneve enabled | All                              | UDP 6081          |
+| Antrea with STT enabled    | All                              | TCP/UDP 1607      |
+| Antrea with GRE enabled    | All                              | UDP 4754          |
+| Antrea with IPSec enabled  | All                              | UDP 500           |
```

Comment:
This is IPsec ESP, no UDP.

Reply:
Hi @jianjuns , I am checking this page https://www.itprotoday.com/strategy/q-what-firewall-ports-should-we-open-make-ipsec-work-through-our-firewalls and should I say it's "IP protocol ID 50 and 51" required for IPsec ESP? UDP port 500 is no needed?

Reply:
As it says, IPsec ESP is "IP protocol 50 and 51", but not UDP (IP protocol 17). However, as we are using strongSwan as IKE, it might require some ports (I think UDP port 500/4500) to be open too.

Reply:
yes, 500 and 4500 are opened after checking them on node VM.

Reply:
UDP port 500 is required for IPsec IKE.
UDP port 4500 is for IPsec NAT-T (AH and ESP).
IP protocol 50 (ESP), 51(AH) for non-NAT only.

Reply:
UDP port 500 is configurable in some platform.

Reply:
@jianjuns  I have tried four tunnels type and ipsec, only one STT tunnel doesn't work, so I can't confirm the 7471 port is used or not.(I didn't see any process listen on it), and others are correct.